src/resolver/Mutation/create.js

```diff
@@ -1,8 +1,7 @@
 export default (entityData = []) => (_, entity) => {
     const newId =
         entityData.length > 0 ? entityData[entityData.length - 1].id + 1 : 0;
-    const newEntity = Object.assign({ id: newId }, entity);
-
+    const newEntity = Object.assign(entity, { id: newId });
```

Comment:
Object.assign mutates the first argument. The correct syntax should be:

```js
Object.assign({}, entity, { id: newId });
```

Reply:
Hello @fzaninotto already updated my PR